docs/reference/migration/migrate_5_0.asciidoc

```diff
@@ -24,7 +24,7 @@ version of Elasticsearch before 2.0.
 =========================================
 
 Indices created in Elasticsearch 1.x or before will need to be reindexed with
-Elasticsearch 2.x in order to be readable by Elasticsearch 5.x. It is not
+Elasticsearch 2.x or 5.x in order to be readable by Elasticsearch 5.x. It is not
```

Comment:
Possessive. I was just thinking it is nice to mention that in order to reindex from 1.x to 5.x you need reindex-from-remote. You can't use regular reindex.

It looks like github ate my attempt to write asciidoc syntax - I've edited the comment.